client/blocks/product-selector/index.jsx

```diff
@@ -66,6 +66,19 @@ export class ProductSelector extends Component {
 		);
 	}
 
+	getPurchaseUpgrade( purchase, product ) {
```

Comment:
If this is taking care of upsells rather than limiting upgrades, let's make sure to make that clear in the code.